### DISCUSSION TITLE
Trying to setup email alerts

### DISCUSSION THREAD

**Discussion by aetherpirate** [Security-Onion-Solutions/securityonion#7164] [Unsupported Versions] [answered]
Looked at https://docs.securityonion.net/en/16.04/email.html
 
so-email does not exist on my installation

I am running Version: 2.3.100

**Comment by IzacFrank** [comment]
Hello,
You can use elastalert for that. You can create yaml files under /opt/so/rules/elastalert/
Please check #7051 I did  for MS Teams and for the email part use the below. For more info https://docs.securityonion.net/en/2.3/elastalert.html?highlight=elastalert

```
alert:
- email

# (required, email specific)
# a list of email addresses to send alerts to

email:
- "receiver@xxx.com"
smtp_host: "x.x.x.x"
smtp_port: 25
# smtp_ssl: true
# smtp_auth_file: '/opt/elastalert/rules/smtp_auth_file.txt'
from_addr: "noreply@yyyyy.com"

```

**Comment by dougburks** [comment, marked as answer]
> Looked at https://docs.securityonion.net/en/16.04/email.html

That is the email documentation for the old Security Onion 16.04 which is now past End Of Life.

Since you're using Security Onion 2.3.100, you'll want to look at the 2.3 documentation:
https://docs.securityonion.net/en/2.3/email.html
